ably/realtime_conn.go

```diff
@@ -15,11 +16,23 @@ var (
 	errQueueing = errors.New("unable to send messages in current state with disabled queueing")
 )
 
+// ConnectionMode is the mode in which the connection is operating
+type ConnectionMode uint
```

Comment:
This and associated constants shouidn't be public.

Reply:
addressed in 71dafc5cc4d75fd14c09e66ad2907a30bc2c08ef